.github/workflows/react-native-metrics.yml

```diff
@@ -19,9 +19,9 @@ jobs:
       with:
         xcode-version: ${{ matrix.xcode }}
     - name: Setup Node.js environment
-      uses: actions/setup-node@v1.4.4
+      uses: actions/setup-node@v2
       with:
-        node-version: '12.x'
+        node-version: '16.x'
```

Comment:
side-comment - Most of our customers will probably be on node 12 or 14.  It usually takes companies a while to upgrade.  LTS is often even numbers.

Reply:
This is because of the BuildProject app requiring npm 7+. We should still work with Node versions as far back as 12.